lighthouse-core/gather/driver.js

```diff
@@ -187,9 +187,10 @@ class Driver {
    * Call protocol methods
    * @param {!string} method
    * @param {!Object} params
+   * @param {boolean=} silent
    * @return {!Promise}
    */
-  sendCommand(method, params) {
+  sendCommand(method, params, silent) {
```

Comment:
yeah regardless of that change the connection is currently hard coded for all messages with an error property to log them even if you don't care about the promise resolution